src/Roc.ts

```diff
@@ -17,7 +17,8 @@ export interface IRocConfig {
 function createAxios(url: string) {
   return axios.create({
     baseURL: url,
-    withCredentials: true
+    withCredentials: true,
+    headers: { Accept: 'application/json' }
```

Comment:
Why do you need this? JSON is supposed to be the default with axios according to the documentation.

Reply:
fixed